rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/controller/admin/GetAllUsersNatsController.kt

```diff
@@ -18,8 +20,17 @@ class GetAllUsersNatsController(
 
     override val parser: Parser<GetAllUsersRequest> = GetAllUsersRequest.parser()
 
-    override fun handle(request: GetAllUsersRequest): GetAllUsersResponse {
-        val userIds: List<String> = adminService.getAllUsers().map(User::telegramUserId)
-        return GetAllUsersResponse.newBuilder().addAllTelegramUserIds(userIds).build()
-    }
+    override fun handle(request: GetAllUsersRequest): GetAllUsersResponse =
+        runCatching {
+            val userTelegramIds: List<String> = getTelegramUserIds()
+            GetAllUsersResponse.newBuilder()
+                .setSuccess(Success.newBuilder().addAllTelegramUserIds(userTelegramIds))
+                .build()
```

Comment:
Maybe make it more Kotlin-like using apply?

Reply:
```kotlin
GetAllUsersResponse.newBuilder().apply {
    successBuilder.telegramUserIdsBuilder.addAll(userTelegramIds)
}.build()
```